app/src/main/java/com/banuba/quickstart_c_api/Image.java

```diff
@@ -0,0 +1,37 @@
+package com.banuba.quickstart_c_api;
+
+import java.nio.ByteBuffer;
+
+public class Image {
+
+    Image() {}
+
+    Image(byte[] image0, byte[] image1, byte[] image2, int w, int h) {
+        if(image0 != null) {
```

Comment:
assert yPlane != null && uPlane != null && vPlane != null

Reply:
For NV12 format vPlane is null, so this assert will cause an error

Reply:
For example:

OffscreenEffectPlayerImage(byte[] yPlane, byte[] uPlane, byte[] vPlane, int width, int height) {
 assert yPlane != null && uPlane != null && vPlane != null;
  initialize(...);}
OffscreenEffectPlayerImage(byte[] yPlane, byte[] uvPlane, int width, int height) { 
assert yPlane != null && uvPlane != null;
 initialize(...);}

private void initialize(byte[] plane0, byte[] plane1, byte[] plane2, int width, int height) {...}